src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -62,6 +64,24 @@ const {
   Deletion,
 } = ReactTypeOfSideEffect;
 
+function transferRef(fiber: Fiber, element: ReactElement<any>) {
+  if (typeof element.ref === 'string' && element._owner) {
+    // Should we check if this string refs is equal to the previous string ref
+    // and reuse it?
```

Comment:
I'm not sure if it is faster or not actually. But we could try attaching string/owner as properties on the function. We only pay for it if you use it.